src/anm/player.js

```diff
@@ -377,6 +377,8 @@ Player.prototype.load = function(arg1, arg2, arg3, arg4) {
             state.happens = C.RES_LOADING;
             player.fire(C.S_CHANGE_STATE, C.RES_LOADING);
             player.fire(C.S_RES_LOAD, remotes);
+            // actually start loading remote resources
+            anim._loadRemoteResources(player);
```

Comment:
The added `check()` is exactly for the case you described - if the resource is already available, be it cache or super-fast loading, then we just go on.

An added bonus is that the movie now starts instantaneously when loaded the second time - it still took some loader spinning before as the RM went through X cycles of `check()` where X is the number of remote resources (and `check()` isn't really the simplest operation as it is).